PullRequest.py

```diff
@@ -24,6 +32,14 @@ def validate_labels(self, required_any, required_all, banned):
             return False
         return True
 
+    def post_status(self, status_json):
+        url = 'https://api.github.com/repos/{full_repo_name}/statuses/{sha}'.format(
+            full_repo_name=self.full_repo_name,
+            sha=self.head_commit)
+        print(url)
+        r = requests.post(url, data=status_json)
```

Comment:
does this need credentials?

Reply:
yea, already added in the most recent commit